src/components/atoms/NavBar/index.tsx

```diff
@@ -1,22 +1,26 @@
 "use client";
```

Comment:
Do we also need this over here? As we're not using any client feature? I always get confused about this
@pedrohbl03